### ISSUE TITLE
commit comment, readme updates

### ISSUE THREAD

**Issue by mwrich4** [PiSupply/PiJuice#85]
Each commit or updated should have brief notes so that changes can be independently verified and tested.  Helpers will do this when good information is provided by all.  Lacking this, it will be an abandoned mess.

**Comment by shawaj** [comment]
We already do this, but thanks anyway

**Comment by mwrich4** [comment]
odd that the firmware change from today has no comment - huh

**Comment by shawaj** [comment]
@mwrich4 - not sure where you are looking but I can see comments...

```
Adding updated firmware V1.2

Alarm fix. 
Initialization fix after firmware update and configuration reset.
```

**Comment by shawaj** [comment]
You can see it here - https://github.com/PiSupply/PiJuice/commit/d2787a952b585126251bcd487c9051a997490ca3

Also discussed in #79 further

**Comment by mwrich4** [comment]
'Alarm fix.'  
Where I work that wouldn't fly .. need to say what was fixed so the verifier/tester can make sure that it works as intended.   
(this is guesswork) Something like:  Alarm fix: USB voltage alarm now correctly responds to voltage too high / voltage too low.  Before voltage too high triggered a fault when above 5.1 volts.

**Comment by shawaj** [comment]
@mwrich4 thanks for your input. we will take it into consideration for future.

**Comment by mwrich4** [comment]
I made edit to last comment.  All I can think of.

**Comment by tvoverbeek** [comment]
Note, this firmware version should not have been posted on github (my private opinion), since it is still under test to fix the problems described in issue #79 
The 1.2 version posted still has problems and introduced a new one (incorrectly reporting its version).
